src/main/java/sirius/biz/translations/MultiLanguageStringHelper.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.translations;
+
+import sirius.biz.importer.ImportHelper;
+import sirius.biz.importer.ImporterContext;
+import sirius.kernel.commons.Strings;
+import sirius.kernel.nls.NLS;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Provides a central helper when importing data for {@link MultiLanguageString} fields.
+ * <p>
+ * Basically we support two ways of importing data into a multi language string field. Either this helper is
+ * configured appropriately (using {@link #setReplaceOnImport(boolean)}, {@link #setImportLanguage(String)}, or
+ * {@link #setDefaultImportLanguage(String)}) or either {@link #createReplacement()} or {@link #createUpdate()}
+ * are used to build {@link MultiLanguageStringValue} objects which are picked up by the
+ * {@link MultiLanguageStringExtender} which will update the fields accordingly.
+ * <p>
+ * For exporting data, {@link #setExportLanguage(String)} and {@link #setDefaultExportLanguage(String)} can be used
+ * to control which values are actually exported.
+ */
+public class MultiLanguageStringHelper extends ImportHelper {
+
+    /**
+     * Represents a builder pattern to setup a new value to be imported into a {@link MultiLanguageString} field.
+     */
+    public static class MultiLanguageStringValue {
+        protected boolean replace;
+        protected Map<String, String> data;
+
+        protected MultiLanguageStringValue(boolean replace) {
+            this.replace = replace;
+        }
+
+        /**
+         * Specifies the new fallback value to use.
+         *
+         * @param value the new fallback value
+         * @return the builder itself for fluent method calls
+         */
+        public MultiLanguageStringValue withFallback(String value) {
+            return withText(MultiLanguageString.FALLBACK_KEY, value);
+        }
+
+        /**
+         * Specifies a value to store for the given language
+         *
+         * @param lang  the language to store the value for
+         * @param value the new value for the given language
+         * @return the builder itself for fluent method calls
+         */
+        public MultiLanguageStringValue withText(String lang, String value) {
+            if (data == null) {
+                data = new HashMap<>();
+            }
+
+            data.put(lang, value);
+
+            return this;
+        }
+    }
+
+    private boolean replaceOnImport;
+    private String importLanguage;
+    private String defaultImportLanguage;
+    private String exportLanguage;
```

Comment:
Do we really need to have pairs for import export, or could simply join/share them?